blueocean-executor-info-plugin/pom.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>3.19</version>
+    </parent>
+    <name>Blue Ocean Executor Info</name>
+    <artifactId>blueocean-executor-info</artifactId>
+    <version>0.2-beta-2-SNAPSHOT</version>
+    <packaging>hpi</packaging>
+    <description>Show what executors are doing what.</description>
+    <url>https://wiki.jenkins-ci.org/display/JENKINS/Amazing+Plugin</url>
+    <properties>
+        <java.level>8</java.level>
+        <jenkins.version>2.121.1</jenkins.version>
+        <node.version>6.2.1</node.version>
+        <npm.version>3.10.10</npm.version>
+        <blueocean.version>1.8.2</blueocean.version>
+    </properties>
```

Comment:
These properties and license can be removed since its coming from parent

Reply:
Done